pkgs/top-level/all-packages.nix

```diff
@@ -39006,4 +39006,6 @@ with pkgs;
   resgate = callPackage ../servers/resgate { };
 
   oversteer = callPackage ../applications/misc/oversteer { };
+
+  s0ix-selftest-tool = callPackage ../tools/system/s0ix-selftest-tool { };
```

Comment:
Keep the file roughly alphabetically sorted, don't just add it at bottom.

Reply:
There is little order in this file, and there are probably two dozen places that would be "roughly alphabetical". I moved it up to a different area anyway.